fjord/__init__.py

```diff
@@ -49,3 +58,11 @@ def setup_vendor_path():
 
 
 setup_vendor_path()
+
+monkeypatch()
```

Comment:
Why do we have to do the `monkeypatch()` call here? Which code path doesn't have monkeypatching?